### DISCUSSION TITLE
Problem with setting up email server

### DISCUSSION THREAD

**Discussion by wisdomlight** [nextcloud/all-in-one#4273] [Questions] [answered]
In Basic settings > Email server
Trying to use my gmail 
selecting either the STARTLS or SSL (with the ports 587 465 respectively)
I get the following error message:
A problem occurred while sending the email. Please revise your settings. (Error: Email could not be sent. Check your mail server log)

Any suggestions?

**Comment by szaimen** [comment, marked as answer]
Hi, please ask this question at https://help.nextcloud.com. Thanks!

**Reply by wisdomlight** [reply]
ok so I found [this answer](https://help.nextcloud.com/t/cant-send-mails-via-nextcloud-basic-settings/160440/5?u=wisdomlight) - which means I need to reach the config.php
when I `docker exec -u www-data -it nextcloud-aio-nextcloud /bin/bash` there is no text editor available
```
059c0a8c4126:/var/www/html/config$ cp config.php config.php.original
059c0a8c4126:/var/www/html/config$ vim config.php
bash: vim: command not found
059c0a8c4126:/var/www/html/config$ vim.tiny config.php
bash: vim.tiny: command not found
059c0a8c4126:/var/www/html/config$ nano config.php
bash: nano: command not found
059c0a8c4126:/var/www/html/config$ apt install vim.tiny
bash: apt: command not found
```
Am I looking at the right place? if I wget or curl vim.tiny or nano - will it cause everything to stop :-) ?

**Reply by szaimen** [reply]
See https://github.com/nextcloud/all-in-one?tab=readme-ov-file#how-to-edit-nextclouds-configphp-file-with-a-texteditor
